doc/articles/features/windows-networking.md

```diff
@@ -9,7 +9,31 @@ uid: Uno.Features.WNetworking
 
 * The `Windows.Networking` namespace provides classes for accessing and managing network connections from your app.
 
-## Checking for internet connectivity
+## Supported features
+
+| Feature        |  Windows  | Android |  iOS  |  Web (WASM)  | macOS | Linux (Skia)  | Win 7 (Skia) | 
+|----------------------------------|-------|-------|-------|-------|-------|-------|-|
+| `GetInternetConnectionProfile`   | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| `NetworkStatusChanged`           | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+
+## Platform-specific
+
+### Android
+
+Android can recognize all values of `NetworkConnectivityLevel`. iOS, macOS and WASM return either `None` or `InternetAccess`.
+
+**Note**: On Android, the `android.permission.ACCESS_NETWORK_STATE` permission is required. It can be added to the application manifest or with the following attribute in the Android platform head:
```

Comment:
```suggestion
The `android.permission.ACCESS_NETWORK_STATE` permission is required. It can be added to the application manifest or with the following attribute in the Android platform head:
```